differential-dataflow/src/trace/wrappers/frontier.rs

```diff
@@ -150,10 +153,8 @@ impl<C: Cursor> Cursor for CursorFrontier<C, C::Time> {
     fn map_times<L: FnMut(Self::TimeGat<'_>, Self::DiffGat<'_>)>(&mut self, storage: &Self::Storage, mut logic: L) {
         let since = self.since.borrow();
         let until = self.until.borrow();
-        let mut temp: C::Time = <C::Time as timely::progress::Timestamp>::minimum();
         self.cursor.map_times(storage, |time, diff| {
-            use crate::IntoOwned;
-            time.clone_onto(&mut temp);
+            let mut temp = C::owned_time(time);
```

Comment:
This might be a slight regression on account of not re-using the `temp` allocation.

Reply:
Done